Procfile

```diff
@@ -36,10 +36,9 @@ celery_scheduler: REMAP_SIGTERM=SIGQUIT newrelic-admin run-program celery beat -
 # to the `store_pulse_{pushes,jobs}` queues for `worker_store_pulse_data` to process.
 # NB: These should not be scaled up to more than 1 of each.
 # TODO: Merge these two listeners into one since they use so little CPU each (bug 1530965).
-pulse_listener_pushes: newrelic-admin run-program ./manage.py pulse_listener_pushes
-pulse_listener_tasks: newrelic-admin run-program ./manage.py pulse_listener_tasks
+pulse_listener_tasks_pushes: newrelic-admin run-program ./manage.py pulse_listener
```

Comment:
This change to merge pulse tasks/pushes seems orthogonal to the filtering by test paths.  Did these commits get mixed in by mistake?